xpad.c

```diff
@@ -568,6 +577,7 @@ static const struct xboxone_init_packet xboxone_init_packets[] = {
 	XBOXONE_INIT_PKT(0x0000, 0x0000, xboxone_fw2015_init),
 	XBOXONE_INIT_PKT(0x045e, 0x02ea, xboxone_s_init),
 	XBOXONE_INIT_PKT(0x045e, 0x0b00, xboxone_s_init),
+	XBOXONE_INIT_PKT(0x045e, 0x0b00, extra_input_packet_init),
```

Comment:
We technically only need to send this to controllers on v5.11 and later firmwares, but older firmwares don't seem to be bothered by this.

I figured it was best to just keep it simple and reuse the existing pattern... even if that results in sometimes sending these bits without any purpose.